sagemaker-keyspaces/SageMaker_Keyspaces_ml_example.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    deleteKeyspace = "DROP KEYSPACE IF EXISTS blog"
</pre><p>Create a more unique name for the keyspace than "blog" or this could cause customers to accidentally delete resources they want to preserve. Preferably include a timestamp, such as <code>DEMO-keyspaces-{timestamp}</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Done.  Changed schema to  blog_{YYYYMMDD} and all corresponding code</p>